src/librustdoc/html/static/settings.js

```diff
@@ -50,5 +66,5 @@
         }
     }
 
-    setEvents();
+    window.addEventListener("DOMContentLoaded", setEvents);
```

Comment:
Why this change?

Reply:
I realized that the script was loaded before the DOM. In itself, it seems to be working but I guess it doesn't hurt to be sure that the DOM is actually there before trying to work on it.